### ISSUE TITLE
Static CSL files (CSV) changing almost hourly, if not every hour?

### ISSUE THREAD

**Issue by cenright** [InternationalTradeAdministration/developerportal#140]
Still testing trying to figure out exactly what is going on - (I think the data is just re-ordered?) - Can't yet confirm exactly how often there are changes or if they are actually relevant for sure yet...  Previously, there were "changes" at most a couple times a day.
Is this just temporary while working our the recent update? 
At least before a "HEAD" method could be used to just check the LastModified Header to see if the file changed - but that broke.
It appears the ContentLength / file size is the same -> MD5 on downloaded file shows different hash.
Going forward, are we expected to download every hour to see if there are any "real" changes?

**Comment by jamesisaacs** [comment]
Hi @cenright. The importers check for new data every hour but there are not frequent updates to the source data. The consolidated.json file lists ```source_last_updated``` for each source at the end of the file. That value reflects when the source data last changed.

```
  "sources_used": [
    {
      "last_imported": "2022-10-06T05:15:00.004987+00:00",
      "source_last_updated": "2022-08-02T16:12:10+00:00",
      "source": "Capta List (CAP) - Treasury Department",
      "import_rate": "Hourly"
    },
    {
      "last_imported": "2022-10-06T05:15:00.004987+00:00",
      "source_last_updated": "2022-10-05T21:35:25+00:00",
      "source": "Denied Persons List (DPL) - Bureau of Industry and Security",
      "import_rate": "Hourly"
    }, ...

**Comment by cenright** [comment]
Thanks...

Perhaps I am missing something?

Assuming it does actually reflect when the list was actually last changed vs just checked against the sources / but data was the same (perhaps ordered differently?), it might be useful at least.

I think the static Json file is 18 Meg now?  downloading just to find out it's the same data?  I'd think ITA would want to provide a way to tell us in advance if we need to download a large chunk of data...

Perhaps an API call to get just the "sources_used" data?

I guess we should be throttling how often we check to see if it changed / at the expense of being behind the latest data?

**Comment by jamesisaacs** [comment]
@cenright Good point. I'll put in a ticket for us to address this issue. It would be very easy for us to make the sources_used information available separately.

**Comment by edsp-david** [comment]
@cenright We have the CSL [sources endpoint](https://developer.trade.gov/api-details#api=consolidated-screening-list&operation=sources) that might work for you.

![image](https://user-images.githubusercontent.com/31735841/196782017-4ab5e8d0-8d4c-4cc4-8bec-5758341342e0.png)

**Comment by cenright** [comment]
> @cenright There is CSL [sources endpoint](https://developer.trade.gov/api-details#api=consolidated-screening-list&operation=sources) that might work for you.
> 
> ![image](https://user-images.githubusercontent.com/31735841/196782017-4ab5e8d0-8d4c-4cc4-8bec-5758341342e0.png)

Thanks will try soon.
